holmes/plugins/toolsets/opensearch/opensearch_utils.py

```diff
@@ -149,24 +149,22 @@ def build_query(
 class BaseOpenSearchToolset(Toolset):
```

Comment:
Please add test that enabled this toolset and make sure it is enabled correctly after init with the relevant config/env vars.